### ISSUE TITLE
EAO is lower than report

### ISSUE THREAD

**Issue by yangwf1** [GengZ/siameseFC-pytorch-vot#3]
I follow the same data preparation procedure and run the code, but only get 0.1801 EAO on VOT17. So i'm wondering is there any other issues that i should notice

**Comment by GengZ** [comment]
Hi, 

here's a quick checklist:
```
1. Environments are all the same as suggested, especially the version.
2. See if earning rate, number of GPU used, batch size are all correct.
3. test more models, e.g. epoch 25 ~ epoch 50, since sometime the performance could vary across different epochs (sometime the fluctuation can be large).
```

**Comment by yangwf1** [comment]
@GengZ  Thank you, and i have another question, why not use all the data in VID but just the val data. Since the code in line #119 in gen_image_crops_VID.py shows that only the val data is included.

**Comment by GengZ** [comment]
Sure all data with label should be used.
That was a mistake there since I crop the training data and val data separately.
